### PR TITLE
Update the BugTracker to beta4

### PR DESCRIPTION
**Pull request by troydai**
/cc @davidfowl @Praburaj @suhasj @muratg @Eilon

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@troydai__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Troy Dai). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

File: src/BugTracker/Startup.cs [hunk 1]
```diff
@@ -13,24 +12,23 @@ public void ConfigureServices(IServiceCollection services)
...
-        //Configure SignalR
+        // Configure SignalR
         app.UseSignalR();
 
-        //Serves static files in the application.
+        // Serves static files in the application.
         app.UseFileServer();
 
-        //Configure WebFx
+        // Configure WebFx
```
Comment (Praburaj) [inline src/BugTracker/Startup.cs]: Mvc?

File: src/BugTracker/api/BugsController.cs [hunk 2]
```diff
@@ -1,10 +1,10 @@
-﻿using BugTracker.Hubs;
+﻿using System.Collections.Generic;
+using System.Linq;
```
Comment (Praburaj) [inline src/BugTracker/api/BugsController.cs]: sort namespaces.

File: src/BugTracker/project.json [hunk 3]
```diff
@@ -6,13 +6,14 @@
...
     "webroot": "wwwroot",
     "dependencies": {
-        "Kestrel": "1.0.0-*",
-        "Microsoft.AspNet.Server.IIS": "1.0.0-*",
-        "Microsoft.AspNet.Mvc": "6.0.0-*",
-        "Microsoft.AspNet.Server.WebListener": "1.0.0-*",
-        "Microsoft.AspNet.Diagnostics": "1.0.0-*",
-        "Microsoft.AspNet.StaticFiles": "1.0.0-*",
-        "Microsoft.AspNet.SignalR.Server": "3.0.0-*"
+        "Kestrel": "1.0.0-beta4-*",
```
Comment (Praburaj) [inline src/BugTracker/project.json]: I think you could just make it `*`. When we merge to master we can freeze the versions. That we we need not change the versions for every release.

File: src/BugTracker/api/BugsController.cs (hunk 2 above)
Comment (troydai) [inline src/BugTracker/api/BugsController.cs]: Hum? Aren't they sorted?

File: src/BugTracker/Startup.cs (hunk 1 above)
Comment (troydai) [inline src/BugTracker/Startup.cs]: Will update.

File: src/BugTracker/api/BugsController.cs (hunk 2 above)
Comment (Praburaj) [inline src/BugTracker/api/BugsController.cs]: ah..Looked at the wrong file. Ignore :smile:

File: src/BugTracker/project.json (hunk 3 above)
Comment (troydai) [inline src/BugTracker/project.json]: We don't know the final beta4 build version yet. The wildcard will be swept once they're decided.

Comment (troydai) [inline src/BugTracker/project.json]: /cc @glennc 

So here's the policy I will follow:

```
dev: 1.0.0-*
release: 1.0.0-<current milestone>-*
master: 1.0.0-<last milestone>
```

Comment (Praburaj) [inline src/BugTracker/project.json]: why not follow the same rest of the other release branch repos follow?

Comment (troydai) [inline src/BugTracker/project.json]: I think other branches are doing wrong.

Comment (troydai) [inline src/BugTracker/project.json]: All right. I'll follow the convention for now.

**Comment by troydai** [comment]
Ping. Ready for sign off?

**Comment by Praburaj** [comment]
Looks good :shipit:

**Pull request merged**